<launch>

<arg name="debug" default="false" />
<arg name="debug_prefix" value="gdb --args" if="$(arg debug)"/>
<arg name="debug_prefix" value="" unless="$(arg debug)"/>

<!-- pose_calibrator node -->
<node name="pose_calibrator" pkg="mcptam" type="pose_calibrator" clear_params="true" output="screen" launch-prefix="$(arg debug_prefix)"> 
  <rosparam command="load" file="$(find mcptam)/groups/$(arg group_name).yaml" />
  <remap from="reset" to="pose_calibrator/reset" />
  <param name="image_transport" type="string" value="raw"/> 
  <param name="pattern_width" type="int" value="5"/>
  <param name="pattern_height" type="int" value="7"/>
  <param name="square_size" type="double" value="0.3"/>
  <param name="finder_max_ssd_per_pixel" type="int" value="200" />
  <param name="kf_distance_mean_diff_fraction" type="double" value="0.0" />
  <param name="kf_adaptive_thresh" type="bool" value="false" />
  <param name="mm_large_point_test" type="bool" value="false" />
  <param name="mm_only_first_cam" type="bool" value="true" />
  <param name="level_zero_points" type="bool" value="true" />
  
  
  <param name="pose_out_file" type="string" value="$(find husky_mcptam)/poses/poses.dat" /> 
  
</node>

<!--
<node pkg="tf" type="static_transform_publisher" name="world_tf" args="0 0 0 -0.5 0.5 -0.5 0.5 world vision_world 100" />
-->

</launch>

